test/test_module_tracker.py

```diff
@@ -25,10 +25,10 @@ class Mod(torch.nn.Module):
             def __init__(self):
                 super().__init__()
                 self.a = Foo()
-                self.b = Foo()
+                self.b = torch.nn.ModuleDict({"nest": Foo()})
```

Comment:
Should we also just add a test for nn.ModuleList just to be safe?

Reply:
Added